isis/src/clipper/apps/jitterfit/jitterfit.xml

```diff
@@ -2,37 +2,89 @@
 
 <application name="jitterfit" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
   <brief>
-    Registers lines from a normal image with lines from the check image.
+    Characterize jitter for rolling shutter images.
   </brief>
   <description>
     <p>
-      This program co-regsisters lines in the main cube (FROM) with the corrisponding line in
-      the checkline cube (FROM2).
-      The sensor line used to collect the checkline is used to center the registration around
-      the same area in the main cube.
+      This program calculates a two sets of polynomial coefficients that characterize the jitter in an
+      image acquired by a rolling shutter instrument. One polynomial for the line direction and on
+      for the sample direction. The ISIS camera model for the instrument can
+      then use the coefficients to adjust for the time dependent jitter.
     </p>
     <p>
-      The main input cube (FROM) is expected to be a cube from a rolling shutter instrument.
-      The checkline cube (FROM2) is expected to contain lines of data read from the same rolling shutter
-      instrument, but read at different times than the coorsponding line in the main cube. The check 
-      cube must also have an associated table containing the sensor line number and time the line was read. 
-      The algorithum co-registers these individual checklines to the main cube and calculates an offset
-      in the line and sample directions. It then fits an Nth order polynomial through the offsets and time to
-      characterize the jitter. The coefficients are then written to the cube labels so the rolling shutter
-      camera model can adjust the look direction of each pixel in the model's detector map.
+      The coefficients are
+      calculated by coregistering lines in the cube with each line in the check line cube, where each
```

Comment:
Maybe update the first instance of "cube" here to "image cube" to explicitly draw the distinction with the "check line cube"